inference-engine/thirdparty/ocv/opencv_hal_avx.hpp

```diff
@@ -48,7 +48,7 @@ inline __m256d _v256_permute2x128(const __m256d& a, const __m256d& b)
 { return _mm256_permute2f128_pd(a, b, imm); }
 
 template<int imm, typename _Tpvec>
-inline _Tpvec v256_permute2x128(const _Tpvec& a, const _Tpvec& b)
+static inline _Tpvec v256_permute2x128(const _Tpvec& a, const _Tpvec& b)
```

Comment:
why static is required here?

Reply:
Here, the static tells the compiler to force this function inline.

Reply:
Thanks!

Reply:
Please resolve discussion as I haven't now such possibility